src/formViews/Welcome.js

```diff
@@ -1,7 +1,7 @@
 import React, { Fragment } from 'react';
 import { Link } from 'react-router-dom';
 
-import imageTimer from './img/timer.png';
```

Comment:
Why are we removing the timer I quite liked that? Has this been agreed to in the group?

Reply:
We can put back in, but with the new bullets, it'll push the CTA below the fold:

![image](https://user-images.githubusercontent.com/7261527/57180042-746fe100-6e7c-11e9-888e-13eae2b7eb43.png)